template.rb

```diff
@@ -135,7 +134,7 @@ def get_content_between(content, string_start, string_end)
 
 # Setup the template root path
 # If the template file is the url, clone the repo to the tmp directory
-template_root = __FILE__ =~ %r{\Ahttps?://} ? remote_repository : __dir__
+template_root = __FILE__.match?(%r{\Ahttps?://}) ? remote_repository : __dir__
```

Comment:
one question: Is `=~` faster than `match?`? or are we trying to be consistent with the other places in the repo?

Reply:
I made the change because Rubocop suggested it :D 
But it turns out:
- `=~` returns `nil` if no match and the matching strings if match.
- `.match?` is a predicate, so it returns `true` or `false` and so is slightly more performant.
- Warning to not mix with `match` (without `?`) that returns a `MatchData` object.

> 💡  [source](https://www.rubydoc.info/gems/rubocop/0.47.1/RuboCop/Cop/Performance/RegexpMatch)

But IMO, it is more a matter of consistency across all the files: The performance gain this template generation is totally negligible compared to the other tasks involved.